satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -615,6 +615,31 @@ read all (
 	where storagenode_storage_tally.interval_end_time >= ?
 )
 
+//--- peer_identity ---//
+
+model peer_identity (
+    key serial_number 
```

Comment:
The key needs to be node_id and serial_number. The serial_number is unique per CA.

Reply:
Afaik there is nothing that verifies or validates certificate serial numbers at all. They should all be random if they were generated with our identity tool but if someone changed a serial number nobody would notice as long as they re-signed the certificate.

Reply:
Yeah, that's fine, but that would be a violation of the spec... so when they do that, it would be fine if we reject their signatures.

Reply:
Ah, sorry... the key should just be `node_id`.

Reply:
in that case how can I make multiple entries for same node_id with different certificates?

Reply:
I thought @jtolds mentioned that it would be fine to just keep the latest identity rather than all of them.

Reply:
got it..

Reply:
updated the changes to update an existing entry